wremnants/CardTool.py

```diff
@@ -802,7 +805,7 @@ def addPOISumGroups(self, gen_axes=None, additional_axes=None, genCharge=None):
         for n in range(2, len(self.datagroups.gen_axes)):
             axes_combinations += [k for k in itertools.combinations(self.datagroups.gen_axes, n)]
         for axes in axes_combinations:
-            logger.debug(f"Add sum group for {axes}")
+            logger.debug(f"Add sum group for {axes}"+("" if genCharge is None else f" with {genCharge}"))
```

Comment:
You can put this all in the same f string if you use ''

Reply:
Done